service/src/main/java/com/inventory/services/EmployeeServiceImpl.java

```diff
@@ -0,0 +1,85 @@
+package com.inventory.services;
+
+import com.inventory.models.Employee;
+import com.inventory.models.Paging;
+import com.inventory.repositories.EmployeeRepository;
+import com.inventory.webmodels.requests.DeleteRequest;
+import com.inventory.webmodels.responses.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class EmployeeServiceImpl implements EmployeeService {
+
+    @Autowired
+    private EmployeeRepository employeeRepository;
+
+    @Override
+    @Transactional
+    public EmployeeResponse getEmployee(String id) {
+       EmployeeResponse response =  new EmployeeResponse();
+       response.setValue(employeeRepository.findById(id).get());
```

Comment:
service only return data from repository, EmployeeResponse should only in webmodel and cannot be included in service layer.
Service return Employee to controller, in controller convert it to EmployeeResponse